yapapi/network.py

```diff
@@ -181,39 +172,39 @@ async def __aexit__(self, *exc_info) -> None:
 
     @property
     def owner_ip(self) -> str:
-        """The IP address of the requestor node within the network."""
+        """Return the IP address of the requestor node within the network."""
```

Comment:
again, these are _properties_, not methods... their docstrings should reflect that...